pkg/webhook/admission/response.go

```diff
@@ -54,7 +56,23 @@ func ValidationResponse(allowed bool, reason string) types.Response {
 	return resp
 }
 
-// PatchResponse returns a new response with json patch.
+// PatchResponseFromRaw takes 2 byte arrays and returns a new response with json patch.
+func PatchResponseFromRaw(original, current []byte) types.Response {
+	patches, err := jsonpatch.CreatePatch(original, current)
+	if err != nil {
+		return ErrorResponse(http.StatusInternalServerError, err)
+	}
+	return types.Response{
+		Patches: patches,
+		Response: &admissionv1beta1.AdmissionResponse{
+			Allowed:   true,
+			PatchType: func() *admissionv1beta1.PatchType { pt := admissionv1beta1.PatchTypeJSONPatch; return &pt }(),
+		},
+	}
+}
+
+// PatchResponse is DEPRECATED. Use PatchResponseFromRaw instead.
```

Comment:
It will be good to highlight why is this being deprecated (may be refer to the issue which explains the roundtripping) ?